rpc_jobs/rpc_upgrades.yml

```diff
@@ -63,19 +82,18 @@
       - "kilo_to_r14.4.1_leap"
     jira_project_key: "RLM"
     jobs:
-      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
 - project:
-    name: "rpc-upgrades-mnaio-newton-head"
+    name: "rpc-upgrades-mnaio-newton-head-pm"
     repo_name: "rpc-upgrades"
     repo_url: "http://github.com/rcbops/rpc-upgrades"
     branches:
      - "master"
     image:
-      - mnaio:
+      - xenial_mnaio:
```

Comment:
Are you using ``xenial`` here to denote the mnaio host, or the guest OS that it's running? How're you informing the MNAIO setup to deploy one distro versus another. I would think it might be best to use this to denote the VM OS so that it's similar in connotation to the AIO setups. That way you can easily inform the MNAIO setup tooling which VM OS to prep too.

Reply:
Good point... I'll change it to reflect the guest OS which will give me one more variable to use.